tests/datadriven/test_asserts.py

```diff
@@ -0,0 +1,105 @@
+"""
+Tests the datadriven assert methods
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import tests.datadriven as datadriven
+
+
+class TestAsserts(unittest.TestCase):
+
+    def setUp(self):
+        self.dataDrivenTest = datadriven.DataDrivenTest()
+
+    def testAssertEqual(self):
+        self.dataDrivenTest.assertEqual(0, 0)
```

Comment:
I presume you mean the lines below the one you tagged.

But no, that's not how the assertRaises statement I created operates.  I find it more convenient to use the with syntax than separating out the arguments from the function.  It's a more general pattern.